django_x509/base/admin.py

```diff
@@ -81,14 +81,14 @@ def get_context(self, data, ca_count=0, cert_count=0):
                 'title': _('Renew selected CAs'),
                 'ca_count': ca_count,
```

Comment:
in this case, since this separation was already done since the begining, we may leave it as is (I mean: we'll keep having two admin.py files, one with the abstract classes and the other ones, even though it's not really necessary, just for backward compatibility`.